internal/providers/azure/azure.go

```diff
@@ -76,57 +76,68 @@ func getClient() retry.Client {
 	return client
 }
 
-func findLease() (*os.File, error) {
+/* godbus doesn't have a way to escape things. This will escape leading numbers, which is all we need */
+func escapeNumber(num int) string {
+	return "_3" + strconv.FormatUint(uint64(num), 10)
```

Comment:
Should work for all numbers, only leading numbers need to be escaped. E.g. 50 would become "_350"